website/content/docs/operators/deployment/schedulers/mesos-local-mac.md

```diff
@@ -0,0 +1,130 @@
+---
+title: Setting up Heron with Mesos Cluster Locally on Mac
+---
+
+This is a step by step guide to run Heron on Mesos cluster locally.
+
+## Get Heron with Mesos Scheduler Implementation
+At the time this article is written, Mesos scheduler is not included in the release version. So you need the latest version of Heron to run it on Mesos. Follow [Compiling](../../../../developers/compiling/compiling) to make sure you can build from the latest version.
```

Comment:
This whole paragraph can be omitted. Assume the docs ship with the version that has support and that people have that version checked out. You can just start the next paragraph with "After compiling, ..."